.pre-commit-config.yaml

```diff
@@ -155,8 +155,7 @@ repos:
     additional_dependencies:
     - types-docutils
     - lxml  # dep of `--txt-report`, `--cobertura-xml-report` & `--html-report`
-    - pytest < 8
```

Comment:
I think `pytest` is still needed, just not capped. MyPy is linting the tests too.

Reply:
Yeah that makes more sense.... getting tired 💤